package.json

```diff
@@ -30,6 +30,8 @@
   },
   "scripts": {
     "build:rtl": "rtlcss style.css style-rtl.css",
+    "build:rtl-esb": "rtlcss assets/css/editor-style-block.css assets/css/editor-style-block-rtl.css",
+    "build:rtl-esc": "rtlcss assets/css/editor-style-classic.css assets/css/editor-style-classic-rtl.css",
```

Comment:
@ntwb Would this be the correct way to create the RTL file for `assets/css/editor-style-classic.css`?

Reply:
Close, I’ll add some functionality so that there is only a single bpm script to run instead of 3